src/support/steps/general.ts

```diff
@@ -84,6 +83,14 @@ Given('activate {string} plugin', async function (this: ICustomWorld, plugin) {
     await this.utils.togglePluginActivation(plugin);
 });
 
+/**
+ * Executes the step to deactivate the WP plugin.
+ */
+Given('deactivate {string} plugin', async function (this: ICustomWorld, plugin) {
+    await this.utils.gotoPlugin();
+    await this.utils.deactivatePlugin(plugin);
```

Comment:
In the future, we'll use WP CLI for this kind of actions.
```suggestion
    await this.togglePluginActivation(plugin, false);
```